pyrobosim/pyrobosim/utils/general.py

```diff
@@ -99,3 +101,30 @@ def replace_special_yaml_tokens(in_text, root_dir=None):
     out_text = out_text.replace("$PWD", root_dir)
 
     return out_text
+
+
+def parse_color(color):
+    """
+    Parses a color input and returns an RGB tuple.
+
+    :param color: Input color as a list, string, or hexadecimal.
+    :type color: list[float] | str
+    :return: RGB tuple in range (0.0, 1.0).
+    :rtype: list[float]
+    """
+    if isinstance(color, list) and len(color) == 3:
+        return color
```

Comment:
really, tuples and lists should both work

Reply:
Current implementation should work with both.